apps/mobile/src/components/loading/fetch-wrapper.tsx

```diff
@@ -1,28 +0,0 @@
-import { Error, FetchState, Loading } from '@/components/loading';
```

Comment:
This was un-used post [this flashlist refactor PR](https://github.com/leather-io/mono/pull/1545). 

That PR also broke the activity loading states so it may return but removing for now as it's not used